### PR TITLE
Css nav

### PR DESCRIPTION
**Pull request by raheraperi**
Fixed nav display issues
Fixed responsive issues
Added in rechart component to home page and styled for all responsive views

### CODE REVIEW

File: server/public/main.css [hunk 1]
```diff
@@ -79,11 +84,11 @@ input {
   margin-left: 0;
 }
 
-.navbar-menu button {
+#login-button {
```
Comment (don-smith) [inline server/public/main.css]: Do we need to use an id here or would a class suffice?

File: client/components/Nav.jsx [hunk 2]
```diff
@@ -9,9 +9,9 @@ const Nav = () => {
           <h2>SW</h2>
         </a>
       </div>
-      <div className='navbar-menu'>
+      <div className='navbarmenu'>
         {/* To be replaced by an Auth */}
-        <Link to='/login' className='button'>login</Link>
+        <Link to='/login' className='button' id='login-button'>login</Link>
```
Comment (don-smith) [inline client/components/Nav.jsx]: Let's use a class here instead of an id unless we really need to.

File: client/components/Home.jsx [hunk 3]
```diff
@@ -7,17 +7,16 @@ import Homebuttons from './Homebuttons'
...
 
-        <div className='column' id='homegraph'>
-          <Route path='/' component={Homegraph} />
-        </div>
+      <div className='column' id='about'>
+        <Route path='/' component={About} />
+        <Route path='/' component={Homebuttons} />
+      </div>
+
+      <div className='column' id='homegraph'>
```
Comment (don-smith) [inline client/components/Home.jsx]: class instead of an id?

File: server/public/main.css [hunk 4]
```diff
@@ -54,16 +54,21 @@ input {
...
-  height: 100vh; 
+  height: 100%; 
 }
 
+.container {
+  position: relative;
+}
+
+
+
```
[the remark this answers is not in the document]
Comment (raheraperi) [reply server/public/main.css]: Done

File: server/public/main.css [hunk 5]
```diff
@@ -94,19 +99,20 @@ input {
...
+.container-main {
+  height: 50%; 
+  margin-top: 4rem;
 }
 
 /* HOMEGRAPH  */
 
-.homegraph {
+.homegraph, #homegraph {
```
[the remark this answers is not in the document]
Comment (raheraperi) [reply server/public/main.css]: Done

File: server/public/main.css [hunk 6]
```diff
@@ -196,3 +204,21 @@ input {
   color: #E8E8E8;
   font-weight: 500;
 }
+
+/* MEDIA QUERIES */
+
+@media only screen and (min-width: 800px) {
+
```
[the remark this answers is not in the document]
Comment (raheraperi) [reply server/public/main.css]: Done

File: server/public/main.css [hunk 7]
```diff
@@ -116,8 +122,9 @@ input {
   color: #7AB0AD;
 }
 
-.main-container {
-  height: 50%; 
+.recharts-wrapper, .recharts-surface, .recharts-legend-wrapper, .recharts-default-legend {
+  width: 100% !important;
```
[the remark this answers is not in the document]
Comment (raheraperi) [reply server/public/main.css]: Leaving here as per convo. Is used to override rechart styling.

File: server/public/main.css (hunk 1 above)
Comment (raheraperi) [reply server/public/main.css]: Removed id

File: client/components/Nav.jsx (hunk 2 above)
Comment (raheraperi) [reply client/components/Nav.jsx]: Removed id

File: client/components/Home.jsx (hunk 3 above)
Comment (raheraperi) [reply client/components/Home.jsx]: Done

**Pull request merged**